cudamapper/src/main.cu

```diff
@@ -450,25 +451,26 @@ int main(int argc, char* argv[])
 
     struct QueryTargetsRange
     {
-        std::pair<std::int32_t, int32_t> query_range;
-        std::vector<std::pair<std::int32_t, int32_t>> target_ranges;
+        claragenomics::cudamapper::IndexDescriptor query_range;
```

Comment:
Optional: I think I would put a `using claragenomics::cudamapper::IndexDescriptor;` at the top of `main()`, to make the following code a tiny bit more readable.

Reply:
My plan is to have the whole `main.cu` in namespace `claragenomcis::cudamapper`. I don't see a reason how it to not be in it. I'm planning to do this in a separate PR

Reply:
ok.